src/task2.1/routes/routes.js

```diff
@@ -15,15 +16,17 @@ router.get('/', (req, res) => {
     res.status(StatusCodes.OK).send(status.REST_AVAILABLE);
 });
 
-router.post('/user', validator.body(userSchema), async (req, res, next) => {
+router.post('/user', validator.body(userSchema), async (req, res) => {
     await userService.createUser(req.body)
         .then(() => {
             res.status(StatusCodes.CREATED).send(status.USER_CREATED);
         })
-        .catch((e) => next(e));
+        .catch((e) => {
+            winstonLogger.error(e.message, { methodName: 'userService.createUser', args: req.body });
```

Comment:
You should pass the error with next method's help to the error handling middleware, but update it with name and params to log before that